src/packet/secret_key.js

```diff
@@ -83,6 +83,13 @@ class SecretKeyPacket extends PublicKeyPacket {
      * @type {Object}
      */
     this.privateParams = null;
+    /**
+     * `true` for keys whose integrity is already confirmed, based on
+     * the AEAD encryption mechanism
+     * @type {Boolean}
+     * @private
+     */
+    this.skipValidation = null;
```

Comment:
Bit nitpicky but this naming sounds a bit like a config option rather than a property of the key, I would name it something like `integrityChecked` or `usedModernAEAD`; that way below it's more obvious why it's safe to check the validation.